src/ebay-section-notice/section-notice.tsx

```diff
@@ -13,14 +13,17 @@ import NoticeContent from '../common/notice-utils/notice-content'
 import { EbayIcon, Icon } from '../ebay-icon'
 import { EbaySectionNoticeFooter } from './index'
 
-export type SectionNoticeStatus = 'general' | 'none' | 'attention' | 'confirmation' | 'information'
+export type SectionNoticeStatus = 'general' | 'none' | 'attention' | 'confirmation' | 'information' | 'education'
 export type Props = ComponentProps<'section'> & {
     status?: SectionNoticeStatus;
```

Comment:
@HenriqueLimas shouldn't it be part of the SectionNoticeStatus ENUM?

Reply:
No, we removed enums some time back